### PR TITLE
remove PrioVR integration

### PR DESCRIPTION
**Pull request by birarda**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: fc2f6d47bbf3dc55fa011545feb5c65f052da788 available at: https://gist.github.com/cc234f4a32d92c5e9825<br/><br/>Installers available for download at:<br>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4352.exe<br>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4352.dmg

File: BUILD_WIN.md [hunk 1]
```diff
@@ -16,16 +16,12 @@ If using Visual Studio 2013 and building as a Visual Studio 2013 project you nee
 
 ####nmake & msbuild
```
Comment (Atlante45) [inline BUILD_WIN.md]: Missed msbuild in the title

**Comment by PhilipRosedale** [comment]
@birarda is this one ready to pull?

**Comment by birarda** [comment]
@PhilipRosedale if successfully builds now, it's ready - @Atlante45

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5a601f3676f51acab408942bf1679495a63de376 available at: https://gist.github.com/5fe8912cfb18a984c678<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4352.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4352.dmg

**Pull request merged**